lld/ELF/ScriptParser.cpp

```diff
@@ -235,6 +236,32 @@ void ScriptParser::readVersionScriptCommand() {
   }
 }
 
+void ScriptParser::readNoCrossRefs(bool to) {
+  expect("(");
+
+  script->noCrossRefLists.push_back({});
+  auto &list = script->noCrossRefLists.back();
+
+  if (to && peek() != ")") {
+    StringRef toSection = next();
+
+    list.toSection = toSection;
+  }
```

Comment:
```suggestion
  if (to && peek() != ")")
    list.toSection = next();
```

Reply:
(The semantics of `next()` are already made clear by the assignment to `list.toSection`; same below.)